src/components/RoomsList.tsx

```diff
@@ -1,12 +1,12 @@
 import { ReactNode } from 'react'
 import styled from 'styled-components/macro'
 
-interface RoomsProps {
+interface Props {
```

Comment:
Ahh, I see you changed it from specific to generic. What is your reason for that?

Reply:
Interface is not exported so the more specific name does not add much